src/coreclr/jit/gentree.cpp

```diff
@@ -5973,6 +5973,7 @@ unsigned Compiler::gtSetEvalOrder(GenTree* tree)
                 switch (tree->AsIntrinsic()->gtIntrinsicName)
                 {
                     case NI_System_Math_Atan2:
+                    case NI_System_Math_FMod:
```

Comment:
@dotnet/jit-contrib Could you please sign-off on this change?

Reply:
this change is safe, it's just slightly increases node's cost. I presume this whole part should be rewritten to use `IsIntrinsicImplementedByUserCall`